app/models/concerns/bulkrax/importer_exporter_behavior.rb

```diff
@@ -17,13 +21,7 @@ def next_import_at
     end
 
     def increment_counters(index)
-      current_importer_run.total_work_entries = if limit.to_i.positive?
-                                                  limit
-                                                elsif parser.total.positive?
-                                                  parser.total
-                                                else
-                                                  index + 1
-                                                end
+      current_importer_run.total_work_entries = index + 1 unless limit.to_i.positive? || parser.total.positive?
```

Comment:
@geekscruff If this line hits the `unless`, then wouldn't `current_importer_run.total_work_entries` (which sounds like an int) be assigned a boolean? Is that what we want?

Reply:
Good spot!

Reply:
Ahh I see, that makes sense